saksstatistikk_ef/src/main/kotlin/no/nav/familie/eksterne/kontrakter/saksstatistikk/Saksstatistikk.kt

```diff
@@ -0,0 +1,52 @@
+package no.nav.familie.eksterne.kontrakter.saksstatistikk
```

Comment:
Burde package være `no.nav.familie.eksterne.kontrakter.ef` som for stonadsstatistikk? 
Når det er så lite i disse modulene burde man nesten kunna ha allt i en og den samme modulen :P (en ef-modul?)